src/client/client.cpp

```diff
@@ -223,16 +223,25 @@ void Client::ProcessEvents( const SystemEvents& events )
 		}
 
 		// Select weapon.
+		if( event.type == SystemEvent::Type::Wheel && event.event.wheel.dy )
+		{
+			if(event.event.wheel.dy > 0.0f) TrySwitchWeaponNext();
+			if(event.event.wheel.dy < 0.0f) TrySwitchWeaponPrevious();
+		}
```

Comment:
_> event.event.wheel.dy_ check is unnecessary here.

Reply:
The check avoids min/max relations  and should probably changed to wheel.dy != 0.0f.
It is possible to send wheel events with wheel.dy == 0.0f